pkg/templates/loader/loader.go

```diff
@@ -26,5 +42,13 @@ func LoadSchema(rootDir, templateName, templateVersion string) (s *types.Templat
 	}
 
 	// Continue with other loaders in the future.
-	return nil, fmt.Errorf("no supported schema found for template %s version %s", templateName, templateVersion)
+	return nil, fmt.Errorf("no supported schema found for template %s", templateName)
 }
+
+type Mode string
+
+const (
```

Comment:
can we make the mode define style as same as https://github.com/seal-io/walrus/blob/main/pkg/apis/server.go#L52-L56?